miqa/core/models/global_settings.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+
+class GlobalSettings(models.Model):
+    """Singleton table that holds global settings that need to be editable at runtime."""
+
+    import_path = models.CharField(max_length=1000, default='', blank=True)
+    export_path = models.CharField(max_length=1000, default='', blank=True)
+
+    def save(self, *args, **kwargs):
+        self.pk = 1
+        super().save(*args, **kwargs)
+
+    @classmethod
+    def load(cls):
+        return cls.objects.get_or_create(pk=1)[0]
```

Comment:
Note that these `save` and `load` methods are all that's needed for the singleton behavior.